xbmc/dialogs/GUIDialogSimpleMenu.cpp

```diff
@@ -25,8 +24,11 @@
 #include "utils/URIUtils.h"
 #include "utils/Variant.h"
 #include "utils/log.h"
+#include "video/VideoDatabase.h"
 #include "video/VideoInfoTag.h"
 
+#include <filesystem/BlurayDirectory.h>
```

Comment:
Jenkins will tell you, that this header belongs elsewhere. We expect headers being sorted.